### PR TITLE
Add support for yaml output to istioctl admin log

### PR DESCRIPTION
**Pull request by my-git9**
**Please provide a description of this PR:**
Add yaml format output for istioctl admin log

The test result:
image

### CODE REVIEW

File: istioctl/cmd/istiodconfig.go [hunk 1]
```diff
@@ -377,7 +377,7 @@ func istiodLogCmd() *cobra.Command {
 	outputFormat := "short"
 
 	logCmd := &cobra.Command{
-		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|yaml]",
+		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|json]",
```
Comment (hanxiaop) [inline istioctl/cmd/istiodconfig.go:380]: It should be better to support YAML format instead of revising the example

Comment (my-git9) [reply istioctl/cmd/istiodconfig.go:380]: OK, I will try it later, thanks

File: istioctl/cmd/istiodconfig.go [hunk 2]
```diff
@@ -155,6 +156,17 @@ func (ga *getAllLogLevelsState) run(out io.Writer) error {
 		}
 		_, err = out.Write(outputBytes)
 		return err
+	case "yaml":
```
Comment (hanxiaop) [inline istioctl/cmd/istiodconfig.go:159]: This is duplicate code with JSON output, you can just do something like `case "json", "yaml"` and if it's yaml, do the rest of the logic.

File: istioctl/cmd/istiodconfig.go [hunk 3]
```diff
@@ -377,7 +389,7 @@ func istiodLogCmd() *cobra.Command {
 	outputFormat := "short"
 
 	logCmd := &cobra.Command{
-		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|yaml]",
+		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|json|yaml]",
```
Comment (hanxiaop) [inline istioctl/cmd/istiodconfig.go:392]: The yaml output support should also be added to the flag usage

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**